common/xml/fpga_architecture.xsd

```diff
@@ -584,6 +584,40 @@
     <xs:attribute name="name" type="xs:string" use="required"/>
   </xs:complexType>
 
+  <xs:complexType name="map">
+    <xs:attribute name="from" type="xs:string" use="required"/>
+    <xs:attribute name="to" type="xs:string" use="required"/>
+  </xs:complexType>
+
+  <xs:complexType name="tile_mode">
+    <xs:choice maxOccurs="unbounded" minOccurs="0">
+      <xs:element name="map" type="map"/>
+    </xs:choice>
+    <xs:attribute name="name" type="xs:string" use="required"/>
+  </xs:complexType>
+
+  <xs:complexType name="equivalent_tiles">
+    <xs:choice maxOccurs="unbounded" minOccurs="0">
+      <xs:element name="mode" type="tile_mode"/>
+    </xs:choice>
+  </xs:complexType>
+
+  <xs:complexType name="tile">
+    <xs:choice maxOccurs="unbounded" minOccurs="0">
+      <xs:element name="equivalent_tiles" type="equivalent_tiles"/>
+      <xs:element name="pinlocations" type="pinlocations"/>
+      <xs:element name="fc" type="fc"/>

```

Comment:
I am unsure if it is possible, but it would be nice to set `fc` as required if there is no `default_fc` defined in the architecture.

Reply:
I think that didn't happen before, so lets leave that for another time.